apps/admin/src/app/workspace/tabs/publishing/workspace-publishing-tab.component.scss

```
@@ -0,0 +1,362 @@
+@use '../workspace-tab.shared.scss';
```

Stylelint (17.13.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)